packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts

```diff
@@ -133,23 +112,7 @@ describeNoCompat("GC Random tests", (getTestObjectProvider) => {
 
     const sleep = async (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
     const seed = Math.random();
-    const random = makeRandom(seed);
-
-    beforeEach(async () => {
-        provider = getTestObjectProvider({
-            syncSummarizer: true,
-        });
-        overrideLogger = new IgnoreErrorLogger(provider.logger);
-        provider.logger = overrideLogger;
-
-        // Create a Container for the first client.
-        mainContainer = await createContainer();
-        const mainDataStore = await requestFluidObject<TestDataObject>(mainContainer, "default");
-        connectedContainers = [];
-        closedContainers = [];
-        testNodes = new Map();
-        testNodes.set(mainDataStore.id, new TestNode(mainDataStore.id, ReferenceState.Root));
-    });
+    const random: Random = makeRandom(seed);
```

Comment:
That part I haven't exactly seen, but now that seems really helpful. There are some cadences we do want to specifically run. For now, this will be manual and exclusively for the GC team. We'll have to figure out the rest once we rollout sweep. Thanks for suggesting that we run all the seeds. I may do that in a different PR. There's a lot of logic I will need to refactor and make more generic.

Reply:
I'll be doing more of the randomization extraction in a different PR